### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by AzizAbdullaev**
- https://www.pivotaltracker.com/story/show/185184818

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
   Route,
   Redirect,
 } from 'react-router-dom';
+import ReactGA from 'react-ga';
 
 import { IndexPage } from './pages/index';
 import { CommunityApplicationPage } from './pages/communtiyapplication';
 //hard code 3 near address to show additional data
 
+const TRACKING_ID = 'G-E1XZYSXM34'; // Google Analytics tracking ID
```
Comment (Megha-Dev-19) [inline src/App.js]: should this ID be visible.?(this repo is public) or should it be hidden? may be we can use env for it

Comment (AzizAbdullaev) [reply src/App.js]: > should this ID be visible.?(this repo is public) or should it be hidden? may be we can use env for it

I added to Netify (prod only)

Comment (Megha-Dev-19) [reply src/App.js]: great, could you also confirm if we have a testnet/staging netlify also, where we need to add this key

**Review by Megha-Dev-19 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (AzizAbdullaev) [reply src/App.js]: > great, could you also confirm if we have a testnet/staging netlify also, where we need to add this key

we only need to keep track of users' actions on the site (prod env). We do not need to keep track of our team's actions (env dev)

**Pull request merged**